src/python_testing/matter_testing_infrastructure/matter/testing/matter_testing.py

```diff
@@ -734,6 +734,40 @@ async def _populate_wildcard(self):
                 None, None, GlobalAttributeIds.FEATURE_MAP_ID), Attribute.AttributePath(None, None, GlobalAttributeIds.ACCEPTED_COMMAND_LIST_ID)]), timeout=60)
             self.stored_global_wildcard = await global_wildcard
 
+    async def cluster_guard(self, endpoint: int, cluster: ClusterObjects.Cluster, skip_step: bool = True):
```

Comment:
given that none of the other guard functions work like this, can you make the skip_step = False condition a different function? ex like... cluster_present or something?